auditbeat/module/file_integrity/fileinfo_posix.go

```diff
@@ -67,6 +68,11 @@ func NewMetadata(path string, info os.FileInfo) (*Metadata, error) {
 		fileInfo.Owner = owner.Username
 	}
 
+	getExtendedAttributes(path, map[string]*string{
+		"security.selinux":        &fileInfo.SELinux,
+		"system.posix_acl_access": &fileInfo.POSIXACLAccess,
```

Comment:
This one is apparently in a binary format instead of a string as I thought -- https://github.com/elastic/beats/issues/36265#issuecomment-1678061936. Can you see what the level of effort would be to convert the value into a human readable format.